scala/scala.bzl

```diff
@@ -645,6 +645,25 @@ def scala_repositories():
     sha256 = "f198967436a5e7a69cfd182902adcfbcb9f2e41b349e1a5c8881a2407f615962",
   )
 
+  native.maven_server(
+    name = "scalac_deps_maven_server",
+    url = "http://bazel-mirror.storage.googleapis.com/repo1.maven.org/maven2/",
+  )
+
+  native.maven_jar(
+    name = "guava",
```

Comment:
have anxiety about this since I'm worried about the name collision. I guess if you collide, it is almost certainly the right jar? But still maybe something like: `rules_scala_guava` might be safer?